extensions/default/src/getSopClassHandlerModule.js

```diff
@@ -110,6 +116,7 @@ const makeDisplaySet = instances => {
     isDynamicVolume,
     dynamicVolumeInfo,
     supportsWindowLevel: true,
+    sortVector: [DEFAULT_SORT_ORDER_NUMBER, index],
```

Comment:
Could you add a comment explaining what's happening here? I think I understand, but it might be helpful for others.